common/Perf/Azure.Test.Perf/PerfTest.cs

```diff
@@ -96,6 +96,16 @@ public virtual Task SetupAsync()
 
         public async Task RecordAndStartPlayback()
         {
+            // Make one call to Run() before starting recording, to avoid capturing one-time setup like authorization requests.
```

Comment:
I wonder if this will nudge the numbers in a good direction as this will be essentially be a warmup step.

Reply:
It's unlikely to change the numbers.  We already do a warmup phase (default 5 seconds), and further this only impacts what is recorded, not what is played back.  The main reason for this change is some scenarios have an extra auth request the first time `Run()` is called, which makes it confusing to see multiple recorded requests in the test-proxy outputs.